config/routes.js

```diff
@@ -1,16 +1,19 @@
 import express from 'express';
 import passport from 'passport';
-
 import index from '../app/controllers/index';
 import questions from '../app/controllers/questions';
 import answers from '../app/controllers/answers';
 import avatars from '../app/controllers/avatars';
 import users from '../app/controllers/users';
 import authorization from '../config/middlewares/authorization';
 import game from '../app/controllers/game';
+import leaderboard from '../app/controllers/game';
+import gameLog from '../app/controllers/game';
 
 const auth = authorization.secureLogin;
 const saveGame = game.createGameData;
+const leaderboard = leaderboard.getLeaderboard;
```

Comment:
Duplicate declaration `leaderboard`. 
You can't declare same variable twice.  You have 
```js
import leaderboard from '../app/controllers/game'
```
and 
```js
const leaderboard = leaderboard.getLeaderboard;
```
You have to change one of them to use a different name.

In fact, you can import `getLeaderboard` directly and use it. I.e.
```js
import { getLeaderboard } from '../app/controllers/game'